Costa Rica Technology Institute  for domain estudiantes.tec.cr

leonardoavs opened the pull request:
Information required by JetBrains:

- Official website: https://www.tec.ac.cr 
- Long-term > 1 year IT related course: Master degree —> https://www.tec.ac.cr/programas-academicos/maestria-computacion
- Proof showing that the university recognizes the domain: https://estudaintes.tec.cr

philipto commented:
@leonardoavs This domain does not belong to a university, nor to a college. I have to close the request without merging. Please ensure that you are familiar with this repository rules located at the bottom of [this page](https://github.com/JetBrains/swot) before submitting a new request. 

The proof which you have submitted does not prove anything, the website looks like a suspicious clone of the official website.

If you have any objections, please email us at support@jetbrains.com.

leonardoavs commented:
Hi @philipto , yes it seems like a copy, but it is not a copy, one is the student website and the other is the university, even you have in this repository other domains for the university, I am currently an student of the master gree for this university and the email that they gave me us leonardoavs@estudiantes.tec.cr, moreover the word estudiantes means students in English. 

Moreover the domain for this will be tec.cr, **estudiantes** will be a subdomain of the doamin and you are not able to have a subdomain in an official domain as the tec.cr, and well below is the information of the domain that you could get from the internet from sites as **https://www.whois.net/**. 

It seems because the estudiantes.tec.cr does not have the https certificate, but if you see it is in the student site, and if you look at the owner of the site could be review in the below image, and you could get this information fro the site https://www.whois.net/, that is a site that let you know who is the owner of the site:

![image](https://user-images.githubusercontent.com/4724922/73080756-f7fab380-3e8b-11ea-8bf6-348e4489826d.png)

domain: tec.cr
registrant: CON-6627
admin-c: CON-6627
**nsset: TEC_CR**
registrar: NIC-REG1
registered: 15.07.2008 02:16:27
changed: 15.03.2018 08:01:19
expire: 15.07.2021

contact: CON-6627
**org: Instituto Tecnologico de Costa Rica**
name: Alexander Valerin Castro
address: Del Estadio Fello Meza 400 Mts al Este
address: Cartago
address: 159-7050
address: Cartago
address: CR
**phone: +506.25509200
fax-no: +506.5515971**
**e-mail: avalerin@itcr.ac.cr**
registrar: NIC-REG1
created: 03.06.2011 22:19:00
changed: 22.08.2019 05:46:44

nsset: TEC_CR
nserver: ns1-08.azure-dns.com
nserver: ns2-08.azure-dns.net
nserver: ns3-08.azure-dns.org
nserver: ns4-08.azure-dns.info
tech-c: CON-6627
registrar: NIC-REG1
created: 13.11.2014 09:56:20
changed: 15.03.2018 08:01:19

I will be trying to reach support@jetbrains.com, but please let me know what site information or what proof do you need, because I gave most of the information that I have.

philipto commented:
@leonardoavs Thank you for the clarification. Yes, you are right: there are some signs of fraud, such as certificate which is not issued for the same domain. However, considering the arguments which you clarified the issue with, I reopened and merged the request. Students of this university are now able to request their free licenses at https://www.jetbrains.com/shop/eform/students. Thank you!
